src/Calculator/Views/UnitConverter.xaml.cpp

```diff
@@ -152,19 +152,19 @@ void UnitConverter::OnOfflineNetworkAccess()
 
 void UnitConverter::SetNormalCurrencyStatus()
 {
-    CurrencySecondaryStatus->Text = L"";
+    VisualStateManager::GoToState(this, NormalCurrencyStatus->Name, false);
```

Comment:
We should still do `CurrencySecondaryStatus->Text = L"";` even if the control will be collapsed

Reply:
I'm happy to make this change, but for my own learning, I'd like to know the reasoning behind it.  If the textblock is not visible, what benefit is there to changing it's text?

Reply:
Went ahead with your other suggestion as that seems like a better fix.